src/index.js

```diff
@@ -56,7 +58,7 @@ class LeaderboardApp {
     const ul = document.createElement('ul');
     ul.classList.add('nameList');
 
-    const names = ['Name: 100', 'Name: 50', 'Name: 20', 'Name: 40'];
+    const names = [];
 
     if (names.length > 0) {
       names.forEach((name, index) => {
```

Comment:
- [Optional]👍 Good job on displaying all the scores! However, there's a small improvement that can make the display even better. It's essential to show the top score at the top of the list to make it more user-friendly. To achieve this, I recommend sorting the scores in descending order before rendering them. By sorting the scores, the highest score will always be displayed at the top of the list, providing a better user experience and making it easier for users to find the top score. Keep up the great work, and happy coding! 🚀💻